### PR TITLE
JBPM-6426: Use of Errai's embedded Wildfly server 10.1.0.Final

### PR DESCRIPTION
**Pull request by romartin**
Hey @manstis @mbarkley @psiroky 
Here is the fix for using the version `10.1.0.Final` of the errai's embedded wildfly server.
Depends on this one https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/550 !!
Thanks!

### CODE REVIEW

**Comment by romartin** [comment]
oops @manstis ! Do you want me to update DMN's stuff in this commit too?? (see [here](https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/555)), just realized right now!

**Comment by romartin** [comment]
@manstis well doing it, fixing dmn too, lemme know if you don't want to! otherwise will update this commit in a bit. Thx!

File: kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-standalone/README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Running the application
         -Djava.util.prefs.syncInterval=2000000
         -Dorg.uberfire.async.executor.safemode=true
         -Dorg.uberfire.nio.git.dir=/tmp/dir
+	-Derrai.dynamic_validation.enabled=true
```
Comment (manstis) [inline kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-standalone/README.md]: Want to stick one more space in here to align?

Comment (romartin) [reply kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-standalone/README.md]: oh yes sorry, well seen! thx

**Comment by romartin** [comment]
@manstis done! All updated, DMN too (tested and the webapp works). Thanks!

**Comment by romartin** [comment]
Jenkins please retest this

**Comment by romartin** [comment]
PS: @manstis just as a reminder - do not forget to ensure your IDE's gwt run config property `errai.jboss.home` is using the wilfdly's version `10.1.0.Final`! :)

**Pull request merged**